src/i18n/locale/en.json

```diff
@@ -240,7 +240,7 @@
         "language_with_code": "language with code: {{code}}"
       },
       "title": {
-        "enabled_languages": "Enabled Languages"
+        "enabled_languages": "Used Languages"
```

Comment:
why? language is an option here

Reply:
https://m1.material.io/style/writing.html#writing-language

Reply:
I didn't find anything relevant to this case, still change this one back.

Reply:
>Write for all levels of readers
Pick common words that are clearly and easily understandable to both beginning and advanced English readers. 

Turn on (better), vs. enable (worse). Consequently, "(on)" is better than "enabled"